### ISSUE TITLE
`NoGlutinConfigs` on latest `master`

### ISSUE THREAD

**Issue by virtualritz** [emilk/egui#2689]
I am using an overlay to `master` on `egui` & `eframe`. I think I last `cargo update`d 2-3 days ago. Everything working dandy.

After updating today my `eframe` app doesn't start any more. I get:
```
Error: NoGlutinConfigs(ConfigTemplate { color_buffer_type: Rgb { r_size: 8, g_size: 8, b_size: 8 }, alpha_size: 8, depth_size: 0, stencil_size: 0, num_samples: None, min_swap_interval: None, max_swap_interval: None, config_surface_types: WINDOW, api: None, transparency: false, single_buffering: false, stereoscopy: None, float_pixels: false, max_pbuffer_width: None, hardware_accelerated: None, max_pbuffer_height: None, native_window: Some(Xlib(XlibWindowHandle { window: 77594627, visual_id: 0 })) })
```

I'm on Kubuntu 22.04.

**Comment by emilk** [comment]
I suspect the recent winit upgrade, but perhaps do a git bisect to make sure?

**Comment by virtualritz** [comment]
I have no idea.
I went back many revs on this repo. Basically until my code broke.
The `NoGlutinConfigs` error persists. At the same time the `egui_demo_app` and the `eframe_template`, which is the base of my app, work fine using `egui:master`.

I didn't change any code, just did a `cargo update` yesterday and that brought on the error. Wtf?

**Comment by virtualritz** [comment]
The full log is:
```
2023-02-08T13:03:08.388051Z DEBUG eframe: Using the glow renderer
2023-02-08T13:03:08.397307Z DEBUG eframe::native::run: Entering the winit event loop (run_return)…
2023-02-08T13:03:08.397353Z DEBUG eframe::native::file_storage: Loading app state from "/home/moritz/.local/share/glyphana/app.ron"…
2023-02-08T13:03:08.399827Z  INFO winit::platform_impl::platform::x11::window: Guessed window scale factor: 1.5    
2023-02-08T13:03:08.399850Z DEBUG winit::platform_impl::platform::x11::window: Calculated physical dimensions: 2122x1102    
2023-02-08T13:03:08.406301Z DEBUG winit::platform_impl::platform::x11::ime: failed to create IME context without preedit support.    
2023-02-08T13:03:08.462010Z DEBUG eframe::native::run: Exiting because of an error
2023-02-08T13:03:08.462044Z DEBUG eframe::native::run: Asking to exit event loop…
2023-02-08T13:03:08.462087Z DEBUG eframe::native::run: Received Event::LoopDestroyed - saving app state…
2023-02-08T13:03:08.462098Z DEBUG eframe::native::run: eframe window closed
Error: NoGlutinConfigs(ConfigTemplate { color_buffer_type: Rgb { r_size: 8, g_size: 8, b_size: 8 }, alpha_size: 8, depth_size: 0, stencil_size: 0, num_samples: None, min_swap_interval: None, max_swap_interval: None, config_surface_types: WINDOW, api: None, transparency: false, single_buffering: false, stereoscopy: None, float_pixels: false, max_pbuffer_width: None, hardware_accelerated: None, max_pbuffer_height: None, native_window: Some(Xlib(XlibWindowHandle { window: 14680067, visual_id: 0 })) })
```

When googling for "failed to create IME context without preedit support" I found #2520.

**Comment by emilk** [comment]
We just merged https://github.com/emilk/egui/pull/2526 - maybe latest `master` has some better log messages for you?

**Comment by virtualritz** [comment]
> We just merged #2526 - maybe latest `master` has some better log messages for you?

That fixed it. Cheers!
Thank you for this mindlowingly awesome crate!

**Comment by emilk** [comment]
Thank @coderedart for actually fixing it too 😆
